Frameworks/Foundation/NSInvocation.mm

```diff
@@ -268,3 +277,44 @@ - (void)invoke {
 }
 
 @end
+
+extern "C" void _NSInvocation_ForwardFrame(
+    void* stret, id self, SEL sel, void* frame, /* out */ _NSInvocationForwardReturnInfo* bridgeOut) {
+    NSMethodSignature* signature = [self methodSignatureForSelector:sel];
+    if (!signature) {
+        const char* types = nullptr;
+        types = sel_getType_np(sel);
```

Comment:
no real reason. :)

---
In reply to: [77266740](https://github.com/Microsoft/WinObjC/pull/951#discussion_r77266740) [](ancestors = 77266740)